.circleci/config.yml

```diff
@@ -0,0 +1,212 @@
+version: 2.1
+
+orbs:
+  node: circleci/node@5.1.0
+  win: circleci/windows@5.0.0
+  coveralls: coveralls/coveralls@2.1.0
+
+commands:
+  install:
+    steps:
+      - run: git config --global core.autocrlf input
+      - node/install:
+          node-version: '18.4.0'
+      - run: nvm use 18.4.0
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ arch }}-{{ checksum "yarn.lock" }}
+            - v1-dependencies-{{ arch }}
+      - run: npm install --global yarn
+      - run: echo "[]" > static/contributors.json
+      - run: yarn install --frozen-lockfile
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ arch }}-{{ checksum "yarn.lock" }}
+  load-gh-token:
+    steps:
+      - run: |
+          curl -X POST "$FIDDLE_SECRETS_SERVICE_ENDPOINT?format=shell" -H "Content-Type: application/json" -d '{"token":"'$CIRCLE_OIDC_TOKEN'"}' >> $BASH_ENV
+
+jobs:
+  lint:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - install
+      - load-gh-token
+      - run: yarn run lint
+      - run: yarn run test:report
```

Comment:
Do we want this here? We weren't using it in the past, and looks to me like it runs the test suite and dumps to a file.